app/server/utils/user.py

```diff
@@ -589,7 +580,8 @@ def proccess_create_or_modify_user_request(
             additional_initial_disbursement,
             organisation.token,
             receive_user=user,
-            transfer_subtype=TransferSubTypeEnum.DISBURSEMENT)
+            transfer_subtype=TransferSubTypeEnum.DISBURSEMENT,
+            automatically_resolve_complete=(additional_initial_disbursement <= current_app.config['MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT']))
```

Comment:
It's possible that
```
additional_initial_disbursement != current_app.config['MAXIMUM_CUSTOM_INITIAL_DISBURSEMENT']
```
makes more sense. AKA you need special permission to do anything other than the default.

Actually I think what u have is fine, but throwing this out there anyways lol